client/src/app/pages/issues/affected-applications/affected-applications.tsx

```diff
@@ -181,7 +184,34 @@ export const AffectedApplications: React.FC = () => {
                     <Th {...getThProps({ columnKey: "name" })} />
                     <Th {...getThProps({ columnKey: "description" })} />
                     <Th {...getThProps({ columnKey: "businessService" })} />
-                    <Th {...getThProps({ columnKey: "effort" })} />
+                    <Th
+                      {...getThProps({ columnKey: "effort" })}
+                      style={{ paddingRight: 0 }}
+                    >
+                      <div style={{ display: "flex", alignItems: "center" }}>
+                        <div>
+                          <span>Effort</span>
```

Comment:
This text will all need to run through i18n. Add entries to translations.json file for the english language.

Reply:
Done, Thanks!